lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -20,11 +20,15 @@
 #include "mlir/IR/Verifier.h"
 #include "mlir/Translation.h"
 #include "llvm/ADT/ScopedHashTable.h"
+#include "llvm/ADT/SetVector.h"
 #include "llvm/ADT/SmallPtrSet.h"
 #include "llvm/ADT/StringExtras.h"
+#include "llvm/ADT/TypeSwitch.h"
 #include "llvm/Support/SourceMgr.h"
 #include "llvm/Support/raw_ostream.h"
 
+#include <iostream>
```

Comment:
I know this is a WIP, but please don't use iostream.  You can use `llvm::errs() <<` instead of cerr.  The issues are explained in the llvm coding standards.

Reply:
Cool! This is what I was looking for (to printf debug what was going on), but couldn't find it. :+1: